bitnami/external-dns/README.md

```diff
@@ -171,9 +171,10 @@ The command removes all the Kubernetes components associated with the chart and
 | `ovh.applicationKey`                          | When using the OVH provider with an existing application, specify the application key. (required when provider=ovh and `ovh.secretName` is not provided.)                    | `""`                      |
 | `ovh.applicationSecret`                       | When using the OVH provider with an existing application, specify the application secret. (required when provider=ovh and `ovh.secretName` is not provided.)                 | `""`                      |
 | `ovh.secretName`                              | When using the OVH provider, it's the name of the secret containing `ovh_consumer_key`, `ovh_application_key` and `ovh_application_secret`. Disables usage of other `ovh`.   | `""`                      |
-| `scaleway.scwAccessKey`                       | When using the Scaleway provider, specify an existing access key. (required when provider=scaleway)                                                                          | `""`                      |
-| `scaleway.scwSecretKey`                       | When using the Scaleway provider, specify an existing secret key. (required when provider=scaleway)                                                                          | `""`                      |
-| `scaleway.scwDefaultOrganizationId`           | When using the Scaleway provider, specify the existing organization id. (required when provider=scaleway)                                                                    | `""`                      |
+| `scaleway.scwAccessKey`                       | When using the Scaleway provider, specify an existing access key. (required when provider=scaleway and `scaleway.secretName` is not provided. )                                                                          | `""`                      |
+| `scaleway.scwSecretKey`                       | When using the Scaleway provider, specify an existing secret key. (required when provider=scaleway and `scaleway.secretName` is not provided.)                                                                          | `""`                      |
+| `scaleway.scwDefaultOrganizationId`           | When using the Scaleway provider, specify the existing organization id. (required when provider=scaleway and `scaleway.secretName` is not provided.)                                                                    | `""`                      |
+| `scaleway.secretName`                              | When using the Scaleway provider, it's the name of the secret containing `scwAccessKey`, `scwSecretKey` and `scwDefaultOrganizationId`. Disables usage of other `scaleway`.   | `""`                      |
```

Comment:
This has to be done into the `values.yaml` and then execute the [readme generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool.